Content.Server/Body/Components/RespiratorComponent.cs

```diff
@@ -9,6 +11,46 @@ namespace Content.Server.Body.Components
     [RegisterComponent, Access(typeof(RespiratorSystem))]
     public sealed partial class RespiratorComponent : Component
     {
+        /// <summary>
+        ///     Gas container for this entity
+        /// </summary>
+        [DataField]
+        public GasMixture Air = new()
+        {
+            Volume = 6, // 6 liters, the average lung capacity for a human according to Google
+            Temperature = Atmospherics.NormalBodyTemperature
+        };
+
+        /// <summary>
+        ///     The gas our entity breathes
+        /// </summary>
+        [DataField]
+        public Gas MetabolizedGas = Gas.Oxygen;
+
+        /// <summary>
+        ///     The gas our entity exhales
+        /// </summary>
+        [DataField]
+        public Gas ByproductGas = Gas.CarbonDioxide;
+
+        /// <summary>
+        ///     Volume of our breath in liters
+        /// </summary>
+        [DataField]
+        public float BreathVolume = Atmospherics.BreathVolume;
```

Comment:
as said on discord, i'm not keen on having explicitly incomplete fallback behaviour that will get obsoleted to better simulate both simplemobs and psychomed mobs. i'd either like to see this have more fleshed out fallback behaviour or i'd like to see it get gutted so that this PR is laser focused on "decouple bodysystem from respiratorsystem"